node/examples/legacy_example.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
pre commit stuff doesnt work if your package.json is not at root. Bug in pre-commit library.